core/include/detray/navigation/navigator.hpp

```diff
@@ -849,7 +855,10 @@ class navigator {
         // Actor flagged cache as broken (other cases of 'no trust' are
         // handeled after volume switch was checked in 'update()')
         if (navigation.trust_level() == navigation::trust_level::e_no_trust) {
-            init(track, navigation, cfg, ctx);
+            // Use overstep tolerance instead of path tolerance
+            const bool use_path_tolerance = false;
```

Comment:
I guess this might lead to actual overstepping being missed (which we may not see because, for example, the "always-init" policy is not checked in the integration test). But this case should also be comparatively rarely hit atm, so lets just move ahead with this for now

Reply:
Ah, no wait, I am reading this wrong. This actually does check the overstepping and the other calls are not?

Reply:
Overstepping check is always on regardless of the boolean input